src/MRA.Pages.Api/Controllers/PagesViewController.cs

```diff
@@ -1,25 +1,58 @@
 using MediatR;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using MRA.Pages.Application.Contract.Page.Commands;
 using MRA.Pages.Application.Contract.Page.Queries;
 using MRA.Pages.Infrastructure.Identity;
 
 namespace MRA.Pages.Api.Controllers;
 
+[Authorize(Policy = ApplicationPolicies.SuperAdministrator)]
 public class PagesViewController(ISender mediator)
     : Controller
 {
-    [Authorize(Policy = ApplicationPolicies.SuperAdministrator)]
     public async Task<IActionResult> Index(GetPagesQuery? query = null)
     {
         var result = await mediator.Send(query ?? new GetPagesQuery());
         ViewBag.PageResponses = result;
         return View();
     }
 
-    [Authorize(Policy = ApplicationPolicies.SuperAdministrator)]
     public IActionResult Create()
     {
         return View();
     }
+
+    [HttpPost]
+    public async Task<IActionResult> Create(CreatePageCommand command)
+    {
+        await mediator.Send(command);
+        return RedirectToAction("Index");
+    }
+
+    [HttpGet]
+    public async Task<IActionResult> Edit(string pageName)
+    {
+        var pageResponse = await mediator.Send(new GetPageQuery
+        {
+            Name = pageName
+        });
+        var model = new UpdatePageCommand
+        {
```

Comment:
it doesn;t looks good. Can you just use View(pageResponse)? If pageResponse nad UpdatePageCommand are the same, you can remove one of them